src/BhudeviS/FindMissingNumber.java

```diff
@@ -0,0 +1,19 @@
+package BhudeviS;
+
+public class FindMissingNumber {
+
+	void FindMissingNumber()
+	{
+		int A[]= {1,2,3,4,5,6,7,8,9};
+		int sum= A[0]+A[1]+A[2]+A[3]+A[4]+A[5]+A[6]+A[7]+A[8];
```

Comment:
Hi @BhudeviSAug19 
This is not appropriate way to calculate sum...
write below logic..

n(n+1)/2;

where n is length of array..